README.md

```diff
@@ -53,13 +53,15 @@ The application built in the tutorial is also available on the [How To GraphQL r
 
 ## Configuration
 
-In your app's `config/environment.js`, configure the URL for the GraphQL API:
+In your app's `config/environment.js`, configure the URL for the GraphQL API. Here, you can also include additional Apollo packages that haven't been included by default. Valid package names can be found [here](https://github.com/apollographql/apollo-link/tree/master/packages)
+
 
 ```js
 let ENV = {
   ...
   apollo: {
     apiURL: 'https://test.example/graphql',
+    include: ['apollo-link-batch-http', 'apollo-link-persisted-queries'],
```

Comment:
Whitelist packages is a approach better and more funded in `ember/addons` ecosystem. 

https://github.com/ember-fastboot/ember-cli-fastboot#whitelisting-packages

Reply:
The addon ecosystem isn't _perfect_ about following these conventions, but the rough guidance I've seen is that config that affects runtime stuff goes in `config/environment.js`, while build-time stuff goes in `ember-cli-build.js` (or `index.js` for addons). Given that, I'd vote in favor of the latter.

Reply:
I agree `build-time` seems a better way, it makes sense to me.